### PR TITLE
[docs][PLAT-7408] Add details of certified Helm charts for OpenShift

### PR DESCRIPTION
**Pull request by bhavin192**
This change adds instructions related to Red Hat certified Helm charts for OpenShift environment.
- Removes the operator based installation from preview as we never had preview releases of operator available.
- Deprecate the operator based installation on stable, it is not maintained and we don't have any users.

Have tested the instructions by following the steps on an OpenShift cluster.

https://deploy-preview-19701--infallible-bardeen-164bc9.netlify.app/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift/
https://deploy-preview-19701--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/install-yugabyte-platform/install-software/openshift/

### CODE REVIEW

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:166]:
```suggestion
- [Accessing and configuring YugabyteDB Anywhere](#access-and-configure-yugabytedb-anywhere)
```

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:241]:
```suggestion
If you don't create the `ClusterRoleBinding`, some container-level metrics like CPU and memory will be unavailable.
```

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:240]:
```suggestion
Creating ClusterRole or ClusterRoleBinding with the cluster-admin role are recommended if your intention is to create clusters across multiple namespaces.

```

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:349]:
```suggestion
Operator-based installation of YugabyteDB Anywhere has been deprecated, consider using the [Red Hat certified Helm charts](#certified-helm-chart-based-installation) instead.
```

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:354]:
```suggestion
- [Installing the Operator](#install-the-operator)
```

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md
Comment (bhavin192) [reply docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:240]: Actually, this sentence shouldn't be here. I will remove it.

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes.md
Comment (bhavin192) [reply docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes.md]: Makes sense, thanks!

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md
Comment (bhavin192) [reply docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:100]: I see, didn't realize the fact that we copy preview to stable when a new version is released. 
>  Is it completely not possible to install a preview version using a helm chart?

We are not certifying the preview versions through Red Hat, we might do it later once we have automation in place. Also, there is some bug in OpenShift which shows only the latest version in the UI, so if we release preview version, users won't be able to install the stable versions. 

How about keeping the instructions in both preview and stable with a small note that only stable versions can be installed for the certified Helm charts?

Comment (ddhodge) [reply docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:100]: That would work

**Review by iSignal — APPROVED** [review]
Looks good!

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md
Comment (bhavin192) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:253]:
```suggestion

If you don't create the `ClusterRoleBinding`, some container-level metrics like CPU and memory will be unavailable.
```

**Comment by bhavin192** [comment]
@ddhodge thank you for the review! I made a small change, can you take a look at it and feel free to merge this if it looks good. Or I will merge it once you approve the PR.

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:109]:
```suggestion
- [Accessing and configuring YugabyteDB Anywhere](#access-and-configure-yugabytedb-anywhere)
```

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:108]:
```suggestion
- [Accessing and configuring YugabyteDB Anywhere](#access-and-configure-yugabytedb-anywhere)
```

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:283]: Section linked doesn't exist

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:284]: Section linked doesn't exist

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:394]: Section linked doesn't exist

**Review by ddhodge — COMMENTED** [review]
Some broken links

**Review by ddhodge — APPROVED** [review]

**Pull request merged**
